dbms/src/Storages/Transaction/RegionPersister.cpp

```diff
@@ -276,6 +276,13 @@ RegionMap RegionPersister::restore(const TiFlashRaftProxyHelper * proxy_helper,
         }
         case PageStorageRunMode::MIX_MODE:
         {
+            // V2 data will be transformed to V3, so use config that more easy to gc
+            config.file_roll_size = PAGE_FILE_SMALL_SIZE;
```

Comment:
Since there could be no more pages written into PS v2, I wonder whether it will work to make the current writable file rotate. So that gc can work to free space.

Reply:
And, can we also use the same strategy in table's PS v2 in `gc()`?  i.e. If we find that a PS v2 already empty, and in MIX mode, then force it to run a writable file rotation, and then GC will delete old page files.

Reply:
Added.